Product/pom.xml

```diff
@@ -556,9 +556,9 @@
         <scope>provided</scope>
       </dependency>
       <dependency>
-        <groupId>javassist</groupId>
+        <groupId>org.javassist</groupId>
         <artifactId>javassist</artifactId>
-        <version>3.4.GA</version>
+        <version>3.20.0-GA</version>
```

Comment:
Since javassist(3.20.0-GA) is loaded by other dependency, we should not exclude this jar as CONNECT requires.  Revert this javassist to 3.4.GA will make ValidationSuite pass in both jdk7 and jdk8 (1.8.0_66).